cristin-person/src/main/java/no/unit/nva/cristin/person/model/nva/Person.java

```diff
@@ -33,6 +33,7 @@
 import static no.unit.nva.cristin.person.model.nva.JsonPropertyNames.NAMES;
 import static no.unit.nva.cristin.person.model.nva.JsonPropertyNames.NATIONAL_IDENTITY_NUMBER;
 import static no.unit.nva.cristin.person.model.nva.JsonPropertyNames.RESERVED;
+import static no.unit.nva.cristin.person.model.nva.JsonPropertyNames.VERIFIED;
 
 @JacocoGenerated
 @JsonPropertyOrder({CONTEXT, ID, TYPE, IDENTIFIERS, NAMES, CONTACT_DETAILS, IMAGE, AFFILIATIONS})
```

Comment:
This is something to consider to add to every DTO

Reply:
Another one on the team said earlier that property order is not important because we do not need it other than that it looks better to read for a human. Just keeping it because it was already there

Reply:
Would love to hear more about your opinion by the way, it might be reasons to use it after all